node/src/command.rs

```diff
@@ -5,14 +5,14 @@ use crate::{
 	service,
 };
 use frame_benchmarking_cli::{BenchmarkCmd, ExtrinsicFactory, SUBSTRATE_REFERENCE_HARDWARE};
-use fennel_node_runtime::{Block, EXISTENTIAL_DEPOSIT};
 use sc_cli::SubstrateCli;
 use sc_service::PartialComponents;
+use fennel_node_runtime::{Block, EXISTENTIAL_DEPOSIT};
 use sp_keyring::Sr25519Keyring;
 
 impl SubstrateCli for Cli {
 	fn impl_name() -> String {
-		"Fennel Node".into()
+		"Substrate Node".into()
```

Comment:
Don't change the name of the node or any Fennel-specific configurations. They're there for a reason.

```suggestion
		"Fennel Node".into()
```

Reply:
darn. i thought i removed all of these references. thanks for catching this one.